ruff.toml

```diff
@@ -0,0 +1,12 @@
+exclude = [
+    "venv",
+    "env",
+    "__pycache__",
+    ".git",
+]
+
+indent-width = 4
+target-version = "py310"
+
+[lint]
+ignore = ["E501", "E226"]
```

Comment:
Can you add comments explaining which lints are ignored?

Reply:
Done.